src/styles/getMuiTheme.js

```diff
@@ -109,6 +109,17 @@ export default function getMuiTheme(baseTheme, muiTheme) {
     },
     paper: {
       backgroundColor: palette.canvasColor,
+      zDepthShadows: [
+        [1, 6, 0.12, 1, 4, 0.12],
+        [3, 10, 0.16, 3, 10, 0.23],
+        [10, 30, 0.19, 6, 10, 0.23],
+        [14, 45, 0.25, 10, 18, 0.22],
+        [19, 60, 0.30, 15, 20, 0.22],
+      ].map(d => {
+        const shadowColor = palette.shadowColor || Colors.fullBlack;
```

Comment:
I don't know what to think about `palette.shadowColor`.
That's kind of an undocumented hidden feature.
That makes me think that we should add it to the `lightBaseTheme`.
(I know I asked you to remove it, sorry :grin:)

@alitaheri @newoga What do you think about using `palette.shadowColor` in this file and adding `shadowColor: Colors.fullBlack` in the `lightBaseTheme.jsx`?